src/renderer/pages/torrent-list-page.js

```diff
@@ -1,6 +1,7 @@
 const React = require('react')
 const prettyBytes = require('prettier-bytes')
 const Checkbox = require('material-ui/Checkbox').default
+const LinearProgress = require('material-ui/LinearProgress').default
```

Comment:
I've been trying to keep the material-ui requires separated from the rest of the requires by a newline